go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/openshift/windows-machine-config-operator
 
-go 1.22.4
+go 1.23.0
 
-toolchain go1.22.7
+toolchain go1.23.5
 
```

Comment:
@mansikulkarni96   please check the replacement below,  no longer needed with go1.23

https://github.com/openshift/windows-machine-config-operator/blob/a77059d3e95517723cc6c7046fafeec8ac6135a6/go.mod#L8

Reply:
thanks will update, not ready for review yet.